includes/Commerce/Orders.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package FacebookCommerce
+ */
+
+namespace SkyVerge\WooCommerce\Facebook\Commerce;
+
+use SkyVerge\WooCommerce\PluginFramework\v5_5_4\SV_WC_Plugin_Exception;
+
+defined( 'ABSPATH' ) or exit;
+
+/**
+ * General Commerce orders handler.
+ *
+ * @since 2.1.0-dev.1
+ */
+class Orders {
+
+
+	/** @var string the fetch orders action */
+	const ACTION_FETCH_ORDERS = 'wc_facebook_commerce_fetch_orders';
+
+	/** @var string the meta key used to store the remote order ID */
+	const REMOTE_ID_META_KEY = '_wc_facebook_commerce_remote_id';
+
+
+	/**
+	 * Finds a local order based on the Commerce ID stored in REMOTE_ID_META_KEY.
+	 *
+	 * @since 2.1.0-dev.1
+	 *
+	 * @param string $remote_id Commerce order ID
+	 * @return \WC_Order|null
+	 */
+	public function find_local_order( $remote_id ) {
+
+		// TODO: implement
+		return null;
+	}
+
+
+	/**
+	 * Creates a local WooCommerce order based on an Orders API order object.
+	 *
+	 * @since 2.1.0-dev.1
+	 *
+	 * @param API\Orders\Order $remote_order Orders API order object
+	 * @return \WC_Order
+	 * @throws SV_WC_Plugin_Exception
+	 */
+	public function create_local_order( $remote_order ) {
+
+		// TODO: implement
+		return null;
+	}
+
+
+	/**
+	 * Updates a local WooCommerce order based on an Orders API order object.
+	 *
+	 * @since 2.1.0-dev.1
+	 *
+	 * @param API\Orders\Order $remote_order Orders API order object
+	 * @param \WC_Order $local_order local order object
+	 * @return \WC_Order
+	 */
+	public function update_local_order( $remote_order, $local_order ) {
```

Comment:
Can declare type for `$local_order` (and `$remote_order` once that class exists I suppose)